docs/multilang/configuring-credential-providers.md

```diff
@@ -7,22 +7,27 @@ However, KCL now provides better extensibility to handle, and be enhanced to han
 This document should help multilang customers configure a suitable `CredentialProvider` (or contribute changes to support a new use case!).
 
 ## Sample Provider Configuration
-DEPRECATED: StsAssumeRoleCredentialsProvider can no longer be constructed in this way:
+
+In a Properties file, an `AWSCredentialsProperty` configuration might look like:
 ```
-AWSCredentialsProvider = StsAssumeRoleCredentialsProvider|<arn>|<sessionName>`
+AWSCredentialsProvider = StsAssumeRoleCredentialsProvider|<arn>|<sessionName> 
```

Comment:
Yeah, I thought of this as well. The downside of this is that it would break backwards compatibility to old multilang config files if we change this from `AWSCredentialsProvider` to `AwsCredentialsProvider` throughout the application. While it would better fit the v2 SDK naming, its just a naming convention change that would likely cause some headaches for users. I'm happy to update this, I just wasn't sure if it was worth the tradeoff.